modules/src/webhook-cli.sh

```
@@ -308,26 +331,129 @@ default_subagent_when() {
+        if [ "$have_include" = 1 ]; then
+          echo "agent-box-webhook: pass --claim OR --include, not both —" \
+               "--claim builds the --include for you; write --include" \
+               "yourself if you need rules --claim cannot express" >&2
+          exit 2
+        fi
+        all="[]"
+        # Unquoted on purpose: claims is a space-separated list this script
+        # built, and a spec cannot contain whitespace (a branch name cannot).
+        for c in $claims; do
+          one="$(claim_clauses "$c")" || exit 2
+          all="$("$JQ" -nc --argjson a "$all" --argjson b "$one" '$a + $b')"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)